compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/pager/Pager.kt

```diff
@@ -466,7 +469,10 @@ object PagerDefaults {
             durationMillis = LowVelocityAnimationDefaultDuration
         ),
         highVelocityAnimationSpec: DecayAnimationSpec<Float> = rememberSplineBasedDecay(),
-        snapAnimationSpec: AnimationSpec<Float> = spring(stiffness = Spring.StiffnessMediumLow),
+        snapAnimationSpec: AnimationSpec<Float> = spring(
+            stiffness = Spring.StiffnessMediumLow,
+            visibilityThreshold = flingBehaviorVisibilityThreshold()
```

Comment:
Is it correct to use "fling" concept in the Pager snap physics? (I don't know, so I am asking :))

Reply:
Seems like yes, the pager animation logic calls `SnapFlingBehavior`